### PR TITLE
chore(readme): fix typo and modify the correct description in chinese…

### PR DESCRIPTION
**Pull request by HaleNing**
HI， When i read Chinese README (README.zh-cn.md) , i find some typo and context error, it cloud maybe make Chinese user confuse. so i fix it .

### CODE REVIEW

File: i18n/README.zh-cn.md [hunk 1]
```diff
@@ -85,12 +82,12 @@ questdb stop  // To stop questdb
...
-- [REST API](https://questdb.io/docs/reference/api/rest/) 监听在端口 `9000`
-- [PostgreSQL](https://questdb.io/docs/reference/api/postgres/) 监听在端口
+- [web 控制台](https://questdb.io/docs/develop/web-console/): 监听在端口 `9000`
+- [REST API](https://questdb.io/docs/reference/api/rest/) :监听在端口 `9000`
+- [PostgreSQL](https://questdb.io/docs/reference/api/postgres/): 监听在端口
   `8812`， 支持 wire 协议，
-- [InfluxDB](https://questdb.io/docs/reference/api/influxdb/) 监听在端口 `9009`,
-  支持 line 协议的高吞吐量数据获取
+- [InfluxDB Line Protocol](https://questdb.io/docs/reference/api/influxdb/): 监听在端口 `9009`,
+  QuestDB 实现了[InfluxDB Line Protocol](https://docs.influxdata.com/influxdb/v1.8/write_protocols/line_protocol_tutorial/) 协议来支持高性能，高吞吐量单向数据插入。
```
Comment (amyshwang) [inline i18n/README.zh-cn.md:90]: 实现了--> maybe 依照？

File: i18n/README.zh-cn.md [hunk 2]
```diff
@@ -85,12 +82,12 @@ questdb stop  // To stop questdb
 
 你可以使用以下接口与 QuestDB 进行交互。
 
-- [web 控制台](https://questdb.io/docs/develop/web-console/) 监听在端口 `9000`
-- [REST API](https://questdb.io/docs/reference/api/rest/) 监听在端口 `9000`
-- [PostgreSQL](https://questdb.io/docs/reference/api/postgres/) 监听在端口
+- [web 控制台](https://questdb.io/docs/develop/web-console/): 监听在端口 `9000`
```
Comment (amyshwang) [inline i18n/README.zh-cn.md:85]: Would you consider removing 监听 from these lines at all?

**Review by amyshwang — CHANGES_REQUESTED** [review]
Thank you for fixing the typos. I have made a couple of suggestions and I hope it makes sense!

**Comment by HaleNing** [comment]
thx for @amyshwang  review, i had modify overview description.

**Review by amyshwang — APPROVED** [review]
Thank you :)

**Pull request merged**